adapters/jixie/jixie.go

```diff
@@ -0,0 +1,155 @@
+package jixie
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type JixieAdapter struct {
```

Comment:
Please name this `adapter`. There is no need to export it with a upper case letter. There is no need to repeat the package name.

Reply:
done in commit 0x3afae48. thank you